src/assertEnvironment.ts

```diff
@@ -1,9 +1,11 @@
 import { spy } from "mobx"
 import { useState } from "react"
 
-if (!useState) {
-    throw new Error("mobx-react-lite requires React with Hooks support")
-}
-if (!spy) {
-    throw new Error("mobx-react-lite requires mobx at least version 4 to be available")
+if ("production" !== process.env.NODE_ENV) {
```

Comment:
Why this condition? I think this is safe to have these checks in production because it would break the whole library anyway with a more cryptic error.

Reply:
cause in production this check cannot have a different result as it would have in non prod (no user input involved or something)?

Reply:
You never know, deployment is hard. It can't hurt to have it there imo.

Reply:
Restored